compiler/tensor.cc

```diff
@@ -101,83 +101,90 @@ Tensor::Tensor(const onnx::TensorProto& xtensor)
         CHECK_EQ(0, xtensor.double_data_size());
         CHECK_EQ(0, xtensor.uint64_data_size());
 
-        switch (dtype_) {
+        int64_t const num_elements = shape.GetTotalSize();
+        switch (dtype) {
```

Comment:
Do we still need this switch? I think we can just use `MakeHostArray(dtype, shape, xtensor.raw_data())`?